src/main/java/milutrock/TaskList.java

```diff
@@ -19,11 +19,11 @@ public Task removeTask(int i) {
     }
 
     public void markTaskAsDone(int i) {
-        tasks.get(i).markDone();
+        this.tasks.get(i).markDone();
     }
 
     public void unmarkTaskAsDone(int i) {
-        tasks.get(i).unmarkDone();
+        this.tasks.get(i).unmarkDone();
     }
 
     public int getNumberOfTasks() {
```

Comment:
In the `TaskList` class, the method `getNumberOfTasks` is well-named and accurately conveys that it returns a count of tasks. This avoids any potential confusion that could arise from shorter or ambiguous names like `taskCount`. Keeping names clear and unambiguous is key to maintainable code.